diff -Nru xzgv-0.3/doc/Makefile xzgv-0.3.new/doc/Makefile
--- xzgv-0.3/doc/Makefile	Tue May  9 23:22:49 2000
+++ xzgv-0.3.new/doc/Makefile	Tue May  9 23:23:44 2000
@@ -39,22 +39,7 @@
 install: ../src/install-info installdirs
 endif
 	install -m 444 xzgv.1 $(MANDIR)
-	install -m 444 xzgv.gz xzgv-?.gz $(INFODIR)
-# Update info `dir' file.
-# Info always uses a dir file in preference to a dir.gz, so we don't use
-# dir.gz unless it's the only game in town.
-ifneq ($(INFO_DIR_UPDATE),no)
-	gzip -d xzgv.gz
-	if [ -f $(INFODIR)/dir.gz -a ! -f $(INFODIR)/dir ]; then \
-	  gzip -d $(INFODIR)/dir.gz; \
-	  ../src/install-info xzgv $(INFODIR)/dir; \
-	  gzip $(INFODIR)/dir; \
-	else \
-	  ../src/install-info xzgv $(INFODIR)/dir; \
-	fi
-	chmod a+r $(INFODIR)/dir
-	gzip xzgv
-endif
+	install -m 444 xzgv.info*.gz $(INFODIR)
 
 ../src/install-info: ../src/install-info.c
 	make -C ../src install-info
diff -Nru xzgv-0.3/doc/xzgv.texi xzgv-0.3.new/doc/xzgv.texi
--- xzgv-0.3/doc/xzgv.texi	Tue May  9 23:22:49 2000
+++ xzgv-0.3.new/doc/xzgv.texi	Tue May  9 23:19:33 2000
@@ -1,6 +1,6 @@
 \input texinfo   @c -*-texinfo-*-
 @c %**start of header
-@setfilename xzgv
+@setfilename xzgv.info
 @settitle The xzgv manual
 @setchapternewpage odd
 @c %**end of header
@@ -23,10 +23,10 @@
 @c This is `ifinfo' to stop texi2{html,roff} seeing it, as they don't
 @c handle it.
 @ifinfo
-@dircategory Graphics Utilities
+@dircategory Graphics Utilities:
 @direntry
-* xzgv: (xzgv). A picture viewer for X, with a thumbnail-based file
-                selector.
+* xzgv: (xzgv).				A picture viewer for X, with
+					a thumbnail-based file selector
 @end direntry
 @end ifinfo
 
